app/javascript/packages/i18n/package.json

```diff
@@ -1,5 +1,31 @@
 {
   "name": "@18f/identity-i18n",
-  "private": true,
-  "version": "1.0.0"
+  "private": false,
+  "version": "1.0.0",
+  "type": "module",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "prepublishOnly": "babel index.ts  --out-dir dist --extensions '.ts,.js'"
+  },
+  "exports": {
+    ".": {
+      "source": "./index.ts",
+      "default": "./dist/index.js"
+    }
+  },
+  "license": "CC0-1.0",
+  "bugs": {
+    "url": "https://github.com/18f/identity-idp/issues"
+  },
+  "homepage": "https://github.com/18f/identity-idp",
+  "peerDependencies": {
+    "react": ">=17"
+  },
```

Comment:
React isn't a dependency is it?

Reply:
resolved with commit 7934fb8a5a662a45720aa2c5081b98f35bbafb00